projects/tuist/features/cache-frameworks.feature

```diff
@@ -16,7 +16,7 @@ Feature: Focuses projects with pre-compiled cached xcframeworks
     And I have a working directory
     Then I copy the fixture ios_app_with_transitive_project into the working directory
     Then tuist warms the cache
-    When tuist focuses the target App
+    When tuist focuses the targets App,FrameworkA-iOS
```

Comment:
```suggestion
    When tuist focuses the targets App, FrameworkA-iOS
```

It might break with the addition of space - if that's the case, I'd rather change the regex in the definition of the step

Reply:
I would do it in a separate PR, as it would require changing all the releated acceptance tests